lib/src/actions/keygen_action.dart

```diff
@@ -93,25 +97,16 @@ class KeygenAction {
     _completer.completeError(error);
   }
 
-  Error? _validateMessage(KeygenMessage message) {
+  bool _validateMessage(KeygenMessage message) {
```

Comment:
what is the context of this change?

Reply:
Previously we validate the message and throws error, we listen to the pairingId which is freshly created during pairing, now we are using uid which is same for all keygen from same device. 
So if a user unpair and try to pair again, then during keygen, if we do not have this, this will listen to the old keygen object and throws error.